frameworks/kafka/tests/test_shakedown.py

```diff
@@ -237,14 +235,172 @@ def test_plan():
     assert service_cli('continue deploy Deployment')
 
 
+@pytest.mark.smoke
 @pytest.mark.sanity
 def test_state():
     assert service_cli('state framework_id')
     tasks = service_cli('state tasks')
     assert tasks
 
 
-# Auxiliary
+# --------- Suppressed -------------
+
+
+@pytest.mark.smoke
+@pytest.mark.smoke
+def test_suppress():
+    dcos_url = dcos.config.get_config_val('core.dcos_url')
+    suppressed_url = urllib.parse.urljoin(dcos_url,
+                                          'service/{}/v1/state/properties/suppressed'.format(PACKAGE_NAME))
+
+    def fun():
+        response = dcos.http.get(suppressed_url)
+        response.raise_for_status()
+        return response.text == "true"
+
+    shakedown.wait_for(fun)
+
+
+
+# --------- Port -------------
```

Comment:
All of these "Port" tests are duplicated below in test_shakedown_ports.py. Please remove one of the duplicates.